src/lib/Scenes/MyCollection/Screens/ArtworkImages/MyCollectionArtworkImages.tsx

```diff
@@ -0,0 +1,84 @@
+import { MyCollectionArtworkImages_artwork } from "__generated__/MyCollectionArtworkImages_artwork.graphql"
+import { MyCollectionArtworkImagesQuery } from "__generated__/MyCollectionArtworkImagesQuery.graphql"
+import { FancyModalHeader } from "lib/Components/FancyModal/FancyModalHeader"
+import OpaqueImageView from "lib/Components/OpaqueImageView/OpaqueImageView"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { renderWithPlaceholder } from "lib/utils/renderWithPlaceholder"
+import { useScreenDimensions } from "lib/utils/useScreenDimensions"
+import { Flex, Spacer } from "palette"
+import React from "react"
+import { ActivityIndicator, ScrollView, View } from "react-native"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+
+interface MyCollectionArtworkImagesProps {
+  artwork: MyCollectionArtworkImages_artwork
+}
+
+const MyCollectionArtworkImages: React.FC<MyCollectionArtworkImagesProps> = ({ artwork }) => {
+  const { images } = artwork
+  // const { navigation: navActions } = AppStore.actions.myCollection
+  const dimensions = useScreenDimensions()
+
+  return images !== null ? (
+    <>
+      <FancyModalHeader>Artwork Images</FancyModalHeader>
+      <ScrollView>
+        {images.map((image: any, index: number) => (
+          <View key={index}>
+            <OpaqueImageView
+              // TODO: figure out if "normalized" is the correct version
+              imageURL={image.url?.replace(":version", "normalized")}
+              height={(dimensions.width / (image.width || 1)) * (image.height || 1)}
+              width={dimensions.width}
+            />
+            <Spacer my={1} />
+          </View>
+        ))}
```

Comment:
It's a bit safer to avoid the `any` here and use the infix `!` operator.

```suggestion
        {images.map((image, index) => (
          <View key={index}>
            <OpaqueImageView
              // TODO: figure out if "normalized" is the correct version
              imageURL={image!.url?.replace(":version", "normalized")}
              height={(dimensions.width / (image!.width || 1)) * (image!.height || 1)}
              width={dimensions.width}
            />
            <Spacer my={1} />
          </View>
        ))}
```